samples/README.md

```diff
@@ -0,0 +1,74 @@
+
+## Data Package
+
+TIDES data packages must include a [`datapackage.json`](https://specs.frictionlessdata.io/data-package/).  Key information to include in [`datapackage.json`](https://specs.frictionlessdata.io/data-package/) includes:
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `title` | A human-readable title. | Required |
+| `name` | Identifier string as a URL-friendly slug. | Required |
+| `description` | Short description of data package. | Recommended |
+| `agency` | Transit agency name. | Recommended |
+| `ntd_id` | ID for the National Transit Database. | Recommended |
+| `profile` | Should be `tabular-data-package` | Required |
+| `licenses` | Should be `[{"name": "Apache-2.0"}]` to be consistent with this repository | Required |
+| `contributors` | Array of data contributors `[{"title": "My Name", "github": "my_handle", "email": "me@myself.com"}]` | Recommended |
+| `maintainers` |  Array of data maintainers `[{"title": "My Name", "github": "my_handle", "email": "me@myself.com"}]` | Recommended |
+| `resources` | Array of data files included in your package, formated as a [`tabular-data-resource`](#data-resource)| Required |
+
+### Data Resource
+
+Key fields for each [`tabular-data-resource`](https://specs.frictionlessdata.io/tabular-data-resource/) are as follows:
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `name` | Short sluggable name used to refer to data in this file. | Required |
+| `path` | Path of the data resource file relative to the `datapackage.json` | Required |
+| `schema` | Data schema to use to valdiate the data resource to | Required |
+| `sources` | Array of data sources formatted as a [`source`](#data-source) | Recommended |
+
+### Data Source
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `title` | Description of the data source. | Required |
+| `component` | What technology component was used to generate this data (directly or indirectly)? Examples include `AVL`, `APC`, `AFC`, etc.  | Recommended |
+| `product` | What product was used to generate this data (directly or indirectly)? | Recommended |
+| `vendor` | What company makes this product? | Recommended |
+
+## Data validation
+
+Data with a valid [`datapackage.json`](#data-package) can be easily validated using the [frictionless framework](https://framework.frictionlessdata.io/), which can be installed and invoke as follows:
+
+```bash
+pip install frictionless
+frictionless validate path/to/your/datapackage.json
+```
+
+### Specific files
+
+Specific files can be validated by running the frictionless framework against them and their corresponding schemas as follows:
+
+```sh
+frictionless validate vehicles.csv --schema https://raw.githubusercontent.com/TIDES-transit/TIDES/main/spec/vehicles.schema.json
```

Comment:
need `--schema-sync` flag here or csv header is ignored and fields are assumed to be in the exact position as defined in the schema, essentially making all fields required.